climakitae/core/data_interface.py

```diff
@@ -444,6 +447,7 @@ def __init__(self):
             geometry=gpd.points_from_xy(self.stations.LON_X, self.stations.LAT_Y),
         )
         self._data_catalog = intake.open_esm_datastore(data_catalog_url)
+        self._warming_levels = read_csv_file(gwl_1850_1900_file, index_col=[0, 1, 2])
```

Comment:
Could we get this into a variable like `_warming_levels_times`? Just for difference between this vs the `warming_levels` attribute that's also used in our codebase